specification/cognitiveservices/Language.AnalyzeText-authoring/models.tsp

```diff
@@ -35,19 +35,19 @@ union ProjectKind {
   /**
    * For building an abstractive summarization models which are able to summarize long documents.
    */
-  @added(Versions.v2023_04_15_preview)
+  @added(Versions.v2024_11_15_preview)
   CustomAbstractiveSummarization: "CustomAbstractiveSummarization",
 
   /**
    * For building an text analytics for health model to identify your health domain data.
    */
-  @added(Versions.v2023_04_15_preview)
+  @added(Versions.v2024_11_15_preview)
   CustomHealthcare: "CustomHealthcare",
```

Comment:
this has been deprecated.

Reply:
I will address this separately as it will trigger breaking changes again.